.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+(Describe the goal of this PR. Mention any related Issue numbers)
+
+## Requirements
+
+You will need to sign the CLA if this is your first time contributing.
+
+Please read the [Contributing guidelines](https://github.com/slackapi/java-slack-sdk/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you agree to those rules.
```

Comment:
Are these supposed to be links to Slack's SDK contribute guideline and code of conduct?